docs/pages/versions/unversioned/sdk/image.mdx

```diff
@@ -5,5 +5,71 @@ packageName: 'expo-image'
 ---
 
 import APISection from '~/components/plugins/APISection';
+import { APIInstallSection } from '~/components/plugins/InstallSection';
+import PlatformsSection from '~/components/plugins/PlatformsSection';
+
+**`expo-image`** is a cross-platform React component that loads and renders images.
+
+Main features:
+
+- Designed for speed
+- Support for many image formats (including animated ones)
+- Disk and memory caching
+- Supports [blurhash](https://blurha.sh), a compact representation of a placeholder for an image
+- Transitioning between images when the source changes (no more flickering!)
+- Implements the CSS [`object-fit`](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit) and [`object-position`](https://developer.mozilla.org/en-US/docs/Web/CSS/object-position) properties (see [`contentFit`](#contentfit) and [`contentPosition`](#contentposition) props)
+- Uses performant [`SDWebImage`](https://github.com/SDWebImage/SDWebImage) and [`Glide`](https://github.com/bumptech/glide) under the hood
+
+<PlatformsSection android emulator ios simulator web />
+
+## Installation
+
+> **Info** `expo-image` is not yet available in Expo Go and Snack.
+
+<APIInstallSection />
+
+## Usage
+
+```tsx
```

Comment:
Since the code snippet below is not using TypeScript, I'd suggest replacing `tsx` with `jsx`. This is for being consistent for adding code languages to snippets inside our docs (however, at the same time, I don't think there will be difference for syntax highlights between JSX and TSX so it's okay to use `tsx` too in this case.).